workflows/cloud-client/.gitignore

```diff
@@ -0,0 +1,2 @@
+sa.json
```

Comment:
Is this file needed?

Reply:
I believe a local service account is needed for local testing.

Reply:
I mean do we need the extra gitignore file